src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -82,6 +82,18 @@ const sidebarWidgets = {
       this.render();
     },
   }),
+  optionsWidget: {
+    template: hbs`{{ displayValue }}{{#unless displayValue}}–{{/unless}}`,
```

Comment:
I think maybe we should add a class that does the `:empty` thing that we can add here as a `className`. That way in the future if we have a widget with multiple values that could be empty the class could be applied within the template.